### DISCUSSION TITLE
environment variable configuration suggestion

### DISCUSSION THREAD

**Discussion by function2-llx** [MIC-DKFZ/nnUNet#861] [Ideas]
Hi there,

In my opinion, since it is recommended to install nnU-Net in a virtual environment, it may be a better idea that related environment variables (required [here](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/setting_up_paths.md) in the documentation) are set within the virtual environment as well, while we're currently suggesting doing this in a global way (by editing `.bashrc` e.g.). 

For conda, there's [official documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#macos-and-linux) for how to do this. For pip with [virtualenv](https://pypi.org/project/virtualenv/), I'm not sure what the best way is, but simply modifying the `venv/bin/activate` script should work, or one could refer to [this question on Stack Overflow](https://stackoverflow.com/questions/9554087/setting-an-environment-variable-in-virtualenv).

Thoughts and suggestions are welcome, thanks in advance!

**Comment by FabianIsensee** [comment]
Hi, I think it is easier to set them globally and we should make the installation process as simple as possible. The names are selected such that they should not interfere with anything else. So I would like to keep it the way it currently is
